### PR TITLE
handle init correctly in kmeans

### PR DESCRIPTION
**Pull request by jbrea**

### CODE REVIEW

File: src/MLJClusteringInterface.jl [hunk 1]
```diff
@@ -292,6 +292,9 @@ Train the machine using `fit!(mach, rows=...)`.
 - `metric::SemiMetric=Distances.SqEuclidean`: The metric used to calculate the
   clustering. Must have type `PreMetric` from Distances.jl.
 
+- `init = :kmpp`: how cluster seeds should be initialized.
+   See [documentation of Clustering.jl](https://juliastats.org/Clustering.jl/stable/kmeans.html#Clustering.kmeans).
```
Comment (ablaom) [inline src/MLJClusteringInterface.jl]:
```suggestion
- `init = :kmpp`: One of the following options to indicate how cluster seeds should be initialized:
   - `:kmpp`: KMeans++ 
   - `:kmenc`: K-medoids initialization based on centrality
   - `:rand`: random
   - an instance of `Clustering.SeedingAlgorithm` from Clustering.jl
   - an integer vector of length `k` that provides the indices of points to use as initial cluster centers.
   See [documentation of Clustering.jl](https://juliastats.org/Clustering.jl/stable/kmeans.html#Clustering.kmeans).
```

**Review by ablaom — COMMENTED** [review]
Looks good, thank you. Could you please add `init` to KMedoids as well 🙏🏾 . The seeding options are the same; only "initial cluster centers" becomes "initial medoids" in the doc-string.

**Comment by jbrea** [comment]
Done.

**Review by ablaom — APPROVED** [review]
Good to go. Thanks @jbrea for this valuable contribution.

**Pull request merged**